baus/subsidies.py

```diff
@@ -617,30 +643,33 @@ def calculate_jobs_housing_fees(policy, year, buildings,
                                                      metadata=metadata)
 
 
-@orca.step()
-def subsidized_office_developer(feasibility, coffer, acct_settings, year,
-                                add_extra_columns_func, buildings, summary):
+#@orca.step()
+def subsidized_office_developer(feasibility, coffer, formula, year,
+                                add_extra_columns_func, buildings,
+                                summary, coffer_acct_name):
 
     # get the total subsidy for subsidizing office
-    total_subsidy = coffer["vmt_com_acct"].\
+    total_subsidy = coffer[coffer_acct_name].\
         total_transactions_by_subacct("regional")
 
     # get the office feasibility frame and sort by profit per sqft
-    feasibility = feasibility.to_frame().loc[:, "office"]
+    
+    feasibility = feasibility.loc[:, "office"]
```

Comment:
Hm, surprising that this doesn't need to be a dataframe to keep using it.

Reply:
It is converted to a dataframe here: https://github.com/BayAreaMetro/bayarea_urbansim/pull/244/files/8dece214fe04667710705fa32cfa7e9f51ccd256#diff-8e32919ed5dab8c6af82aeeb375599e2981a9ac4427a7132bf89d899c7499616R1219